### PR TITLE
feat: update musl to 1.2.5

### PR DESCRIPTION
**Pull request by smira**
Update patchset.

### CODE REVIEW

File: musl/patches/handle-aux-at-base.patch [hunk 1]
```diff
@@ -0,0 +1,48 @@
+https://git.alpinelinux.org/aports/plain/main/musl/handle-aux-at_base.patch
```
Comment (frezbo) [inline musl/patches/handle-aux-at-base.patch:1]: kbuild broke?

**Review by frezbo — APPROVED** [review]

File: musl/patches/handle-aux-at-base.patch (hunk 1 above)
Comment (smira) [reply musl/patches/handle-aux-at-base.patch:1]: what?

**Review by TimJones — APPROVED** [review]

File: musl/patches/handle-aux-at-base.patch (hunk 1 above)
Comment (frezbo) [reply musl/patches/handle-aux-at-base.patch:1]: i meant if some of our build got broken without this patch?

**Pull request merged**